string shipment_type
nist_gear/Product part
int32 priority
